### PR TITLE
[#IOCIT-357]  Add integration tests

### PR DESCRIPTION
**Pull request by gquadrati**
#### List of Changes

- Add IT suite to function

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: __integrations__/environments/env.base [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+REDIS_URL=redis
+REDIS_TLS_ENABLED=false
+
+FUNCTION_PORT=7071
+USE_FALLBACK=false
+FF_TYPE=none
+
+STORAGE_CONN_STRING=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://storage-account:10000/devstoreaccount1;QueueEndpoint=http://storage-account:10001/devstoreaccount1;TableEndpoint=http://storage-account:10002/devstoreaccount1;
+DF_STORAGE_CONN_STRING=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://storage-account:10000/devstoreaccount1;QueueEndpoint=http://storage-account:10001/devstoreaccount1;TableEndpoint=http://storage-account:10002/devstoreaccount1;
```
Comment (arcogabbo) [inline __integrations__/environments/env.base:14]: do we need this?

**Review by arcogabbo — APPROVED** [review]

File: __integrations__/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    ports:
+      - ${FUNCTION_PORT}:7071
+    links:
+      - cosmosdb
+      - storage-account
+    depends_on:
+      - functions-node-18
+
+  testagent:
+    image: node:16-alpine
```
Comment (fabriziopapi) [inline __integrations__/docker-compose.yml:48]: why node 16 here?

**Review by fabriziopapi — APPROVED** [review]
LGTM, only a minor comment.

File: __integrations__/docker-compose.yml (hunk 2 above)
Comment (gquadrati) [reply __integrations__/docker-compose.yml:48]: This is the image used for calling tests, so it's not a problem. However I'll updated it, thanks.

File: __integrations__/environments/env.base (hunk 1 above)
Comment (gquadrati) [reply __integrations__/environments/env.base]: No, thanks. Removed in [[#IOCIT-357] remove unused variable](https://github.com/pagopa/io-functions-lollipop/pull/9/commits/1c0e47cac14e46233c98e210561641bd7006e709)

File: __integrations__/docker-compose.yml (hunk 2 above)
Comment (gquadrati) [reply __integrations__/docker-compose.yml]: Done in [[#IOCIT-357] update node-version of it tests image](https://github.com/pagopa/io-functions-lollipop/pull/9/commits/2534de5eaa8695fbb08181c6e739534740b235ea)

**Pull request merged**